accounts/abi/bind/util_test.go

```diff
@@ -125,10 +125,7 @@ func TestWaitDeployedCornerCases(t *testing.T) {
 	// Create a transaction to an account.
 	code := "6060604052600a8060106000396000f360606040526008565b00"
 	tx := types.NewTransaction(0, common.HexToAddress("0x01"), big.NewInt(0), 3000000, gasPrice, common.FromHex(code))
-	tx, err := types.SignTx(tx, types.LatestSignerForChainID(big.NewInt(1337)), testKey)
-	if err != nil {
-		t.Fatalf("Failed to sign transaction: %s", err)
-	}
```

Comment:
is this to reduce diffs? I'm pretty sure I've seen one of these unhandled errors caused trouble in tests. So I'd be willing to keep this

Reply:
It's geth code so we should align with it.
In the libevm branch, we did change this to be an error check, I suggested to revert it since it was out of scope / no obvious reason, to which we agreed let's check the error it doesn't hurt - but I did not realise it was geth code. Anyway, I don't think this would fail really 🤷